### PR TITLE
Cleanup of many SHOULDs

### LINKED ISSUE #111: Check SHOULD requirements for ROQ receivers
This is probably premature, but I noticed 

> To send RTP/RTCP packets over QUIC streams, a sender MUST open a
> new unidirectional QUIC stream. Streams are unidirectional because there is no
> synchronous relationship between sent and received RTP/RTCP packets. A peer that
> receives a bidirectional stream with a flow identifier that is associated with
> an RTP or RTCP stream, **SHOULD** stop reading from the stream and send a
> STOP\_SENDING frame with the application protocol error code set to
> ROQ\_STREAM\_CREATION\_ERROR.

As I noted in comments on PR #91, that's a fundamental violation of the protocol ("MUST open a new unidirectional QUIC stream"). If that's a MUST, ISTM that we need to either make detecting bidirectional ROQ connections a MUST, or explain what would be likely to happen if the receiver just decides to read from the stream and - what - never write to it? or start writing to it, in spite of the stream violating the protocol? 

But this is actually just a reminder that we need to look at SHOULDs for receivers, and make sure that it makes sense for a receiver to ignore the SHOULD action, and explain at least some of the likely consequences if the receiver does so. 

As long as we're still writing significant normative text, it may make sense to wait and check all the SHOULDs at once, later in the process. As long as we don't forget.

### PR DESCRIPTION
**Pull request by SpencerDawkins**
closes #111

### CODE REVIEW

File: draft-ietf-avtcore-rtp-over-quic.md [hunk 1]
```diff
@@ -526,11 +524,11 @@ To send RTP/RTCP packets over QUIC streams, a sender MUST open at least one new
 RoQ uses unidirectional streams, because there is no
 synchronous relationship between sent and received RTP/RTCP packets. A peer that
 receives a bidirectional stream with a flow identifier that is associated with
-an RTP or RTCP stream, SHOULD stop reading from the stream and send a
+an RTP or RTCP stream, MUST stop reading from the stream and send a
```
Comment (mengelbart) [inline draft-ietf-avtcore-rtp-over-quic.md:527]: I wonder if this should lead to closing the connection because it is a protocol violation? We make that change in a different PR though.

**Review by mengelbart — APPROVED** [review]
Thanks, I think this is ready to be merged.

File: draft-ietf-avtcore-rtp-over-quic.md (hunk 1 above)
Comment (SpencerDawkins) [reply draft-ietf-avtcore-rtp-over-quic.md:527]: Actually, you're right - we talked about that, but somehow it didn't make the transition into our shared notes from our call. Let's wait to merge until I can update the PR (I'm having odd Windows problems right now, so that may take a day or two).

Comment (SpencerDawkins) [reply draft-ietf-avtcore-rtp-over-quic.md]: @mengelbart - I think I got this one right. Please let me know! 

If I did, we should be able to merge the PR.
